docs/discover/field-statistics.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[show-field-statistics]]
+== Explore the fields in your data
+
+beta::[]
+
+Explore the fields in your data with the *Field statistics* view in *Discover*.
+You can answer questions such as:
+
+* What does the latency
+look like when one of the containers is down on a Sunday?
+* Is the field type and format specified in the data view appropriate
+for the data and its cardinality?
+
+This example explores the fields in
+the <<gs-get-data-into-kibana, sample web logs data set>>, or you can use your own data.
+
+. Open *Stack Managment > Advanced Settings*, search for *Show field statistics*,
+and turn on the setting.
+
+. Open *Discover*, expand the {data-source} dropdown,
+and select *kibana_sample_data_logs*.
+
+. If you don’t see any results, expand the time range, for example, to *Last 7 days*.
+
+. Click *Field statistics*.
++
+The table summarizes how many documents in the sample
+for the selected time period contain each field,
+the number of distinct values, and the distribution.
++
+[role="screenshot"]
+image::images/field-statistics-view.png["Field statistics view in Discover showing a summary of document data."]
+
+. Expand the `hour_of_day` field.
++
+For numeric fields, *Discover* provides the
```

Comment:
Since the UI is clearly labeled for DOCUMENT STATS, min, median, max, TOP VALUES, and DISTRIBUTION, do you need to list them here? Especially since you included a screenshot? 

I like how you included `Use this chart to get a better idea of how the values in the data are clustered.`